### DISCUSSION TITLE
Add copy to clipboard function when text highlighted in Manifold reader.

### DISCUSSION THREAD

**Discussion by jamie-reece** [ManifoldScholar/manifold#3752] [Ideas]
The ability to natively copy text in Manifold appears overridden by its other functions (at least in Firefox), so might make sense to include a 'copy' function in the options menu to make up for this (I guess depending on cross-browser behaviour).

**Comment by zdavis** [comment]
Yep, this was an oversight on our part, which has been resolved in v9.
